src/components/app.tsx

```diff
@@ -605,10 +607,24 @@ class AppContainer extends React.Component<AppProps, AppState> {
         }
     }
 
+    hasReachedRecordingLimit = () => {
+        const { sensorRecordings } = this.state;
+        const { singleReads, displayType } = this.props;
+        const singleReadBarChart = singleReads && displayType === "bar";
+
+        if (singleReadBarChart && sensorRecordings[0].data.length >= 6) {
```

Comment:
I'm wondering if it would be good to have a global constant for the number of bar chart samples (6) as it is used in other files also.

Reply:
@dougmartin I think that makes sense. Do the latest changes around that look OK?